docs/content/faq.md

```diff
@@ -65,7 +65,7 @@ tar --gzip -cf "backup-$(date +%F).tar.gz" config data
 Then to restore `./config` and `./data` folders from your backup file:
 
 ```bash
-tar -xf backup-date.tar.gz
+tar --gzip -xf backup-date.tar.gz
```

Comment:
[`--gzip` hint isn't needed](https://askubuntu.com/a/398336) for extraction.

Reply:
You are right. But it doesn't hurt if you ask me. I prefer calling tools with explicit parameters, especially in documentation. Feel free to remove it 👍